src/other/utility/utility-en.hbs

```diff
@@ -116,76 +116,96 @@
+<h4>Code sample</h4>
 <pre><code>&lt;p class="<strong>pl-2</strong> bg-primary max-content"&gt;Suspendisse faucibus nisl at consectetur.&lt;/p&gt;</code></pre>
 
 <h3><code>px-2</code></h3>
 <p>Set a left and right padding of 5px.</p>
-<h4>working example:</h4>
+<h4>Working example</h4>
 <p class="px-2 bg-primary max-content">Suspendisse faucibus nisl at consectetur.</p>
-<h4>Code sample:</h4>
+<h4>Code sample</h4>
 <pre><code>&lt;p class="<strong>px-2</strong> bg-primary max-content"&gt;Suspendisse faucibus nisl at consectetur.&lt;/p&gt;</code></pre>
 
 <h3><code>pr-2</code></h3>
 <p>Set a right padding of 5px.</p>
-<h4>working example:</h4>
+<h4>Working example</h4>
 <p class="pr-2 bg-primary max-content">Suspendisse faucibus nisl at consectetur.</p>
-<h4>Code sample:</h4>
+<h4>Code sample</h4>
 <pre><code>&lt;p class="<strong>pr-2</strong> bg-primary max-content"&gt;Suspendisse faucibus nisl at consectetur.&lt;/p&gt;</code></pre>
 
 <h3><code>pt-4</code></h3>
 <p>Set a top padding of 30px.</p>
-<h4>working example:</h4>
+<h4>Working example</h4>
 <p class="pt-4 bg-primary max-content">Suspendisse faucibus nisl at consectetur.</p>
-<h4>Code sample:</h4>
+<h4>Code sample</h4>
 <pre><code>&lt;p class="<strong>pt-4</strong> bg-primary max-content"&gt;Suspendisse faucibus nisl at consectetur.&lt;/p&gt;</code></pre>
 
 <h3><code>py-4</code></h3>
 <p>Set a top and bottom padding of 30px.</p>
-<h4>working example:</h4>
+<h4>Working example</h4>
 <p class="py-4 bg-primary max-content">Suspendisse faucibus nisl at consectetur.</p>
-<h4>Code sample:</h4>
+<h4>Code sample</h4>
 <pre><code>&lt;p class="<strong>py-4</strong> bg-primary max-content"&gt;Suspendisse faucibus nisl at consectetur.&lt;/p&gt;</code></pre>
 
 <h3><code>pb-4</code></h3>
 <p>Set a bottom padding of 30px.</p>
-<h4>working example:</h4>
+<h4>Working example</h4>
 <p class="pb-4 bg-primary max-content">Suspendisse faucibus nisl at consectetur.</p>
-<h4>Code sample:</h4>
+<h4>Code sample</h4>
 <pre><code>&lt;p class="<strong>pb-4</strong> bg-primary max-content"&gt;Suspendisse faucibus nisl at consectetur.&lt;/p&gt;</code></pre>
 
 <h2 id="positioning">Positioning</h2>
 
 <h3><code>position-relative</code></h3>
 <p>Set an element in a relative position. This was created in support of the <code>stretched-link</code> utility. For a better visualisation effect, refer to the <a href="#sl">streched-link utility section</a> below.</p>
-<h4>working example:</h4>
-<p class="position-relative">This is is a pragraph relatively positioned.</p>
-<h4>Code sample:</h4>
-<pre><code>&lt;p class="<strong>position-relative</strong>"&gt;This is is a pragraph relatively positioned.&lt;/p&gt;</code></pre>
+<h4>Working example</h4>
+<p class="position-relative">This is a paragraph relatively positioned.</p>
+<h4>Code sample</h4>
+<pre><code>&lt;p class="<strong>position-relative</strong>"&gt;This is a paragraph relatively positioned.&lt;/p&gt;</code></pre>
+
+<h3 id="d-flex"><code>d-flex</code></h3>
```

Comment:
How should these descriptions/examples be handled when both a generic class and a breakpoint-specific version have been backported?

Just for context, I sent in #9426 a day before this PR. Among other things, it proposes adding ``d-sm-flex`` - with a different description and example than what this PR is pitching for the ``d-flex`` class.